src/openforms/registrations/contrib/objects_api/api/views.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import authentication, permissions, views
+
+from openforms.api.views import ListMixin
+
+from ..client import get_objecttypes_client
+from .serializers import ObjecttypeSerializer, ObjecttypeVersionSerializer
+
+
+@extend_schema_view(
+    get=extend_schema(
+        tags=["registration"],
+    ),
+)
+class ObjecttypesListView(ListMixin, views.APIView):
+    """
+    List the available Objecttypes.
+    """
+
+    authentication_classes = (authentication.SessionAuthentication,)
+    permission_classes = (permissions.IsAdminUser,)
```

Comment:
This threw me off but apparently it's applied consistently through the codebase. Perhaps we need a `AdminUIAccessMixin` to properly centralize this. I had expected that there were additional permissions required to check that you can actually edit forms instead of this being accessible to any admin user :thinking: